### ISSUE TITLE
Copy madani v2

### ISSUE THREAD

**Issue by benomaire** [quran/quran.com-frontend-next#408]
This may be related to https://github.com/quran/quran.com-frontend-next/pull/393
At the moment, if I copy text while showing madani v2, I can see it on LibreOffice for example after manually changing the font to QCF2xxx depending on the page. Note that I get line breaks between words, but that is another issue.
But copying across LibreOffice apps preserves the font info. So, if we can embed that font info in the copied text from next.quran.com, it might be a good option. 
In that case we may add an option: `Copy to LibreOffice` and display a reminder for the user to install fonts on their system (with a link perhaps?)

**Comment by tifa2UP** [comment]
Wa alaikum alsalam, this is indeed being worked on in #393. Once the PR is merged, the functionality should be there iA

**Comment by muhajirdev** [comment]
PR is merged, we should be able to copy madaniv2 now @benomaire . Thanks for reporting this :)

**Comment by benomaire** [comment]
@muhajirdev
I am not sure if I explained my point clear enough. I want something extra than what you have just implemented: To copy madani v2 as is, not convert it to unicode. Of course I will not paste as plain text, but to LibreOffice for example.
I raised the issue since the code is still fresh, and it might be easier to implement this "complex" feature now.
I have two use cases in mind now:
1. for people writing articles for printing, and wanting to have madani v2 exactly.
2. for people making videos, so effectively quran.com could be a good alternative to https://qurancliphelper.com/


So, I don't object the current behaviour, but i suggest adding an extra MIME type that can be recognized by libreoffice. The following links maybe useful:

- https://stackoverflow.com/questions/3571179/how-does-x11-clipboard-handle-multiple-data-formats
- https://github.com/Hi-Angel/scripts/blob/master/print_clipboard_content.py
- https://hluk.github.io/CopyQ/

I appreciate your point of view though, and that unicode plain text is probably the best option for most users.
